### PR TITLE
doc: add ZBOSS banner CSS and manifest

### PR DESCRIPTION
**Pull request by greg-fer**
Added CSS styling for ZBOSS banner in nrfxlib.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/_static/css/nrfxlib.css [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+ width: 200%;
+ height: 200%;
+ right: 0;
+ top: 40%;
+ left: 20%;
+ transform: translate(-50%, -50%);
+}
+
+.yellow-box {
+ background: #ffaa54;
```
Comment (gmarull) [inline doc/_static/css/nrfxlib.css:75]: not sure if it's the color you want exactly, but consider using `var(--docset-color)`

File: west.yml [hunk 2]
```diff
@@ -103,7 +103,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 4a060cdb789b8e8eec025f7164d77322ad0cc2a5
```
Comment (gmarull) [inline west.yml:106]: why is this required? commit does not give any hint

File: doc/_static/css/nrfxlib.css [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+}
```
Comment (gmarull) [inline doc/_static/css/nrfxlib.css:76]: missing NL

File: west.yml (hunk 2 above)
Comment (greg-fer) [reply west.yml:106]: This is to pull in the following PR, which is related: https://github.com/nrfconnect/sdk-nrfxlib/pull/581

File: doc/_static/css/nrfxlib.css (hunk 1 above)
Comment (greg-fer) [reply doc/_static/css/nrfxlib.css:75]: I'm using an image in https://github.com/nrfconnect/sdk-nrfxlib/pull/581 that has almost the same color as the docset, so wanted to go for something brighter.

File: doc/_static/css/nrfxlib.css [hunk 4]
```diff
@@ -1,3 +1,80 @@
...
+ padding: 0 10px;
+ display: block;
+ width: 450px;
+ overflow:hidden;
+ transition: all 0.35s ease;
+ box-sizing: border-box;
+ line-height: 50px;
+ vertical-align:middle;
+ position:relative;
+ margin-left: 0;
```
Comment (gmarull) [inline doc/_static/css/nrfxlib.css:17]: pls format CSS using two spaces indent (as root entry)

File: doc/_static/css/nrfxlib.css (hunk 1 above)
Comment (greg-fer) [reply doc/_static/css/nrfxlib.css:75]: OK, added a different "zboss-box" and changed this one to the default docset color, to match "blue-box" in nrf.css.

File: doc/_static/css/nrfxlib.css (hunk 4 above)
Comment (greg-fer) [reply doc/_static/css/nrfxlib.css]: And done, in a separate commit.

**Review by gmarull — APPROVED** [review]

**Review by maciejpietras — APPROVED** [review]

File: west.yml [hunk 5]
```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 22d60f831a43f988fbb9c4475dd4a5ae657930aa
+      revision: pull/581/head
```
Comment (rlubos) [inline west.yml:107]: Please rebase and update SHA: `c1fe5a41c45f41adcfc008fe0ae2a4265e148d81`

Comment (greg-fer) [reply west.yml]: Rebased and updated SHA.

**Review by rlubos — APPROVED** [review]
